db/migrations/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  extends: "lob/migrations", 
```

Comment:
You may need to remove this trailing comma so that this files valid JSON!

Reply:
done